config.example.wikimedia.yaml

```diff
@@ -81,45 +82,62 @@ templates:
                   - name: default.group.local
                     domains: /./
 
-      /{module:page_revisions}:
+      /{module:page_revisions}: &wp/sys/page_revisions
         x-modules:
             - name: page_revisions
               version: 1.0.0
               type: file
 
-      /{module:key_rev_value}:
+      /{module:key_value}: &wp/sys/key_value
```

Comment:
Why do we need to add this module to `default-sys` domain? It's used only by method, which's only in `global-sys` domain. May be better to inline it there?